content/08-mongodb/10-mongodb-transactions.mdx

```diff
@@ -0,0 +1,246 @@
+
+The benefits of multi-document transactions are tailored for systems that move funds around like banking applications or payment processing, supply chain or shipping systems where ownership of a good is transferred, or in e-commerce. These examples are generally packaging changes together across multiple streams and require strong consistency with the all or nothing approach transactions offer. 
+
+## How do you use transactions?
+
+MongoDB provides two [APIs](https://docs.mongodb.com/manual/core/transactions/#transactions-api) to use transactions. The first is the core API which has similar syntax to relational databases. The second, the callback API, is the recommended approach to using transactions in MongoDB.
+
+The comparison of the two APIs is best summarized in the following table:
+
+| Core API | Callback API |
+| -------- | -----------  |
+| Requires an explicit call to start and to commit the transaction| Starts a transaction, executes the specified operations, and commits (or aborts on error) |
+| Does not automatically incorporate error-handling logic for `TransientTransactionError` and `UnknownTransactionCommitResult` but instead grants the ability to integrate custom error handling. | Automatically incorporates error-handling logic for `TransientTransactionError` and `UnknownTransactionCommmitResult`. |
+| Requires explicit logical session to be passed to API for the specific transaction. | Requires explicit logical session to be passed to API for the specific transaction. |
+
+<PrismaOutlinks>
+
+If you are looking to get started working with MongoDB and Prisma, checkout our [getting started from scratch guide](https://www.prisma.io/docs/getting-started) or how to [add to an existing project](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/mongodb-typescript-mongodb).
+
+</PrismaOutlinks>
+
+### Creating a transaction session
+
+Transactions are typically written and executed from external applications via one of the API methods with the application language's appropriate [MongoDB driver](https://docs.mongodb.com/drivers/).
+
+For the sake of demonstration, we'll walk through the creation of a transaction via the MongoDB shell. We'll work with an example database and collection to help conceptualize how transactions work in practice.
+
+**Note**: If a transactions session runs for more than 60 seconds, MongoDB will automatically abort the operation. This occurs because transactions are typically conducted in applications that work automatically rather than a person issuing commands in the shell. The following steps are for conceptualizing a transaction session from start to finish.  
```

Comment:
I was just curious after reading ahead about whether the 60 seconds starts from the `startSession()` call or the `startTransaction()` call.  I'm not terribly familiar with these, so I wasn't sure which one "transaction session" is referring to.  I thought I might as well mention it.